samples/nrf_desktop/src/modules/ble_adv.c

```diff
@@ -95,6 +95,7 @@ static enum state state;
 
 static struct k_delayed_work adv_update;
 static struct k_delayed_work sp_grace_period_to;
+static u8_t cur_identity = BT_ID_DEFAULT; /* We expect zero */
```

Comment:
Missing dot ?

Reply:
on purpose

Reply:
treated as short comment instead of sentence. should I add it?